gengokit/middlewares/templates/endpoints.go

```diff
@@ -5,9 +5,20 @@ package middlewares
 
 import (
 	"github.com/go-kit/kit/endpoint"
+	svc "{{.ImportPath -}} /generated"
 )
 
-func InjectEndpointMiddlewares(in endpoint.Endpoint) endpoint.Endpoint {
+// WrapEndpoint will be called individually for all endpoints defined in
+// the service. Implement this with the middlewares you want applied to
+// every endpoint.
+func WrapEndpoint(in endpoint.Endpoint) endpoint.Endpoint {
```

Comment:
I have not thought of any great ones yet, but I am going to throw a couple out to get the brain juice flowing.
```
WrapQualifiedEndpoints(...

WrapSelectedEndpoints(...

WrapSpecifiedEndpoints(...

WrapIndividualEndpoints(...
```

From my perspective, these functions should be named in a useful way for the user of truss, the names should describe what the user would get if they filled the function with code. Rather than naming them for what they do. 

Though that is just me, that was the pattern I was going for. 

What about this:
1. Move `func WrapEndpoint(` to `server_main.go`
2. Change the body of `WrapEndpoint` to:
```
func WrapEndpoint(in endpoint.Endpoint) endpoint.Endpoint {
    return middlewares.WrapAllEndpoints(in)
}
```
3. Add `WrapAllEndpoints` to replace `WrapEndpoint`. This way, that function actually does get get called on all Endpoints.